openIMIS/openIMIS/git_scripts.py

```diff
@@ -0,0 +1,112 @@
+import git
+import json
+import os
+
+
+from os import path
+from django.conf import settings
+from openIMIS.openimisapps import openimis_apps
+
+
+def create_release_branches_backend(version):
+    """
+       function to create release branches 'release/<version>'
+       for all backend modules presented in openimis.json
+    """
+    output_messages = []
+    # modules = openimis_apps()
+    # TODO temporary value for modules for test
+    modules = ['calculation_rule-fs_income_percentage', 'report', 'policyholder', 'contribution_plan', 'calculation']
```

Comment:
Is this fixed assignment of values still required?

Reply:
@dborowiecki Yes, this is for test purposes made by me intentionally to run this script only on chosen modules . But as I mentioned - the mechanism to obtain modules names from openimis.json is already implemented, Later if we test this scrips and be sure it works I will remove those lines with 'modules' reaasigment.